aks-node-controller/go.mod

```diff
@@ -9,16 +9,25 @@ require (
 	github.com/fsnotify/fsnotify v1.8.0
 	github.com/google/go-cmp v0.7.0
 	github.com/stretchr/testify v1.10.0
-	google.golang.org/protobuf v1.35.2
+	google.golang.org/protobuf v1.36.5
```

Comment:
we need to configure dependabot to look at go.mod's other than the root...

Reply:
it doesn today. I think it only cares if there are CVEs